defaults.config.yaml

```diff
@@ -3,10 +3,17 @@ host: 0.0.0.0
 # The port to listen on.
 port: 3000
 # The directory in which to store the LevelDB database.
-databaseDataDirectory: db
-# The directory in which to store uploaded file assets.
-fileDataDirectory: files
+databasePlugin: LevelDB
+databaseConfig:
+  databaseDataDirectory: db
+fileStoragePlugin: awsS3Storage
```

Comment:
This broke our tests since our test environments don't have configured S3 credentials.